### PR TITLE
Update for US Government Authority Instance

### PR DESCRIPTION
**Pull request by brandwe**
For @weijjia

### CODE REVIEW

File: src/src/com/microsoft/aad/adal/Discovery.java [hunk 1]
```diff
@@ -137,10 +137,10 @@ private void addValidHostToList(URL validhost) {
     private void initValidList() {
         // mValidHosts is a sync set
         if (sValidHosts.size() == 0) {
-            sValidHosts.add("login.windows.net");
-            sValidHosts.add("login.microsoftonline.com");
-            sValidHosts.add("login.chinacloudapi.cn");
-            sValidHosts.add("login.cloudgovapi.us");
+            sValidHosts.add("login.windows.net"); // Microsoft Azure Worldwide - Used in validation scenarios where host is not this list
```
Comment (RandalliLama) [inline src/src/com/microsoft/aad/adal/Discovery.java]: Could you add to the comment that this is deprecated?

Comment (RandalliLama) [inline src/src/com/microsoft/aad/adal/Discovery.java]: @brandwe ^

File: src/src/com/microsoft/aad/adal/Discovery.java [hunk 2]
```diff
@@ -137,10 +137,10 @@ private void addValidHostToList(URL validhost) {
...
         // mValidHosts is a sync set
         if (sValidHosts.size() == 0) {
-            sValidHosts.add("login.windows.net");
-            sValidHosts.add("login.microsoftonline.com");
-            sValidHosts.add("login.chinacloudapi.cn");
-            sValidHosts.add("login.cloudgovapi.us");
+            sValidHosts.add("login.windows.net"); // Microsoft Azure Worldwide - Used in validation scenarios where host is not this list 
+            sValidHosts.add("login.microsoftonline.com"); // Microsoft Azure Worldwide
+            sValidHosts.add("login.chinacloudapi.cn"); // Microsoft Azure China
+            sValidHosts.add("login-us.microsoftonline.com"); // Microsoft Azure US Government
```
Comment (addev-ashish) [inline src/src/com/microsoft/aad/adal/Discovery.java]: Why don't you add the German one here as well. Just trying to prevent 2 one line code reviews.

**Comment by weijjia** [comment]
Also, @brandwe Please submit the PR to dev

**Comment by weijjia** [comment]
@brandwe I'm going to close this one. Send another to dev, and we start again. :)

**Pull request closed without merging**